sharpy/postproc/aerogridplot.py

```diff
@@ -106,8 +106,9 @@ def run(self, online=False):
         # TODO: Create a dictionary to plot any variable as in beamplot
         if not online:
             for self.ts in range(self.ts_max):
-                self.plot_body()
-                self.plot_wake()
+                if self.data.aero.timestep_info[self.ts] is not None:
```

Comment:
When would we have `None`s in `aero.timestep_info`?

Reply:
When you use the cleanup timesteps postprocessor, the old time steps are set to None to save RAM memory. I really need that because I run out of memory quite quickly. 

If you use the postprocessors "online" (inside DynamicCoupled, for example) there is no problem because you just write the last one. However, if you use them "offline" (in the "flow" variable of SHARPy), you may try to write time steps that were previously cleaned by the "CleanUp" postprocessor.

Reply:
Sounds good, thanks!